src/components/progress-indicator/bl-progress-indicator.stories.mdx

```diff
@@ -0,0 +1,102 @@
+import { html } from 'lit';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import { styleMap } from 'lit/directives/style-map.js';
+import { unsafeHTML } from 'lit/directives/unsafe-html.js';
+import { Meta, Canvas, ArgsTable, Story, Preview, Source } from '@storybook/addon-docs';
+
+<Meta
+  title="Components/Progress Indicator"
+  component="bl-progress-indicator"
+  argTypes={{
+    size: {
+      control: {
+        type: 'select',
+        options: ['medium', 'large']
+      },
+      type: 'string'
+    },
+    max: {
+      control: 'text',
+      type: 'string'
+    },
+    value: {
+      control: 'text',
+      type: 'string'
+    },
+    failed: {
+      control: 'boolean',
+    },
+  }}
+/>
+
+export const ProgressIndicatorTemplate = (args) => html`
+<bl-progress-indicator
+    size='${ifDefined(args.size)}'
+    max='${ifDefined(args.max)}'
+    value='${ifDefined(args.value)}'
+    failed='${ifDefined(args.failed)}'
```

Comment:
```suggestion
    ?failed=${args.failed}
```

Reply:
What is the different between `ifDefined` and `?` expression?

Reply:
`?` is used for boolean attributes. If you don't use it result becomes `failed="true"` or `failed="false"`